<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QMainWindow" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mastermind Lobby</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="accessibleName">
    <string>MainWindow</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="MainVLayout" stretch="3,0,6,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="HeaderFrame">
        <property name="accessibleName">
         <string>Header</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="Title">
           <property name="font">
            <font>
             <family>Lato Light</family>
             <pointsize>36</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="accessibleName">
            <string>Title</string>
           </property>
           <property name="text">
            <string>Mastermind</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ControlsHFrame">
        <property name="accessibleName">
         <string>ControlsFrame</string>
        </property>
        <layout class="QHBoxLayout" name="ControlsHLayout" stretch="4,5,4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="ConnectToServer">
           <property name="accessibleName">
            <string>LobbyButton</string>
           </property>
           <property name="text">
            <string>Spela online</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PlayOffline">
           <property name="accessibleName">
            <string>LobbyButtonCenter</string>
           </property>
           <property name="text">
            <string>Spela själv</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CreateServer">
           <property name="accessibleName">
            <string>LobbyButton</string>
           </property>
           <property name="text">
            <string>Skapa server</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="OutputVLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QTextEdit" name="Output">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Yu Gothic UI Semilight</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yu Gothic UI Semilight'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Fixedsys';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="OutputControlsHLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QLineEdit" name="Input">
            <property name="placeholderText">
             <string>Skriv något..</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="InputSend">
            <property name="accessibleName">
             <string>OutputButton</string>
            </property>
            <property name="text">
             <string>Skicka</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ClearOutputButton">
            <property name="accessibleName">
             <string>OutputButton</string>
            </property>
            <property name="text">
             <string>Rensa</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="FooterFrame">
        <property name="accessibleName">
         <string>FooterFrame</string>
        </property>
        <layout class="QHBoxLayout" name="RulesHLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="FooterHLayout" stretch="0,0,1">
           <item>
            <widget class="QPushButton" name="RulesButton">
             <property name="accessibleName">
              <string>FooterButton</string>
             </property>
             <property name="text">
              <string>Regler</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ContactButton">
             <property name="accessibleName">
              <string>FooterButton</string>
             </property>
             <property name="text">
              <string>Kontakt</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="AboutLabel">
             <property name="accessibleName">
              <string>About</string>
             </property>
             <property name="text">
              <string>Kod och design av Fredrik Engstrand </string>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
